nl/guides/new-course/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cursus aanmaken
+description: "Tutorial: cursus aanmaken"
+---
+
+# Cursus aanmaken
+
+Als lesgever kan je een nieuwe cursus aanmaken waar je studenten zich voor kunnen registreren. Dit doe je door op de knop `CURSUS AANMAKEN` te drukken in de rechterbovenhoek van het cursusoverzicht.
+
+![create course](./staff.courses_new_link.png)
+
+Om een nieuwe cursus aan te maken, zijn er twee opties. Ofwel vertrek je vanaf een reeds bestaande cursus, ofwel maak je een volledige nieuwe cursus aan.
+
+![new course menu](./staff.course_new_options.png)
+
+Druk op `Dodona` in de [navigatiebalk](../for-students#navigatiebalk) om het aanmaken van de cursus te annuleren.
+
+Als je vertrekt vanaf een bestaande cursus, dan moet je deze selecteren in de tabel. Via de zoekbalk kan je filteren om snel de gewenste cursus te vinden. Klik het bolletje in de linkerkolom aan om je keuze te bevestigen.
```

Comment:
add: via de link knop aan de rechter kant van de tabel open je de betreffende cursus in een nieuwe venster. Zo kan je snel controleren of dit de juiste cursus is.